src/amo/components/Header/styles.scss

```diff
@@ -142,11 +142,22 @@
 }
 
 .Header-SectionLinks {
+  align-items: baseline;
   grid-column: 1 / span 4;
   grid-row: 3 / 4;
   margin: 16px auto 1px;
+  min-height: 28px;
   z-index: 4;
 
+  .DropdownMenu-button-text {
+    overflow: visible;
```

Comment:
I'm not sure if this might make @kumar303's white square re-appear.

Reply:
`overflow` should be fine. The weird square was only appearing with `overflow-x` (but I'm not sure why).